manifests/init.pp

```diff
@@ -360,7 +360,7 @@
   $repo                           = 'main',
   $repo_source                    = undef,
   $repo_key_id                    = 'EE15CFF6AB6E4E290FDAB681A20F259AEB9C94BB',
-  $repo_key_source                = 'http://repositories.sensuapp.org/apt/pubkey.gpg',
+  $repo_key_source                = 'https://sensu.global.ssl.fastly.net/apt/pubkey.gpg',
```

Comment:
@ghoneycutt https with `sensu.global.ssl.fastly.net` is OK. 

We need to be careful with https on `repositories.sensuapp.org`, as the cert being provided on that CDN map uses SNI (subjectAltName, not supported by all package managers) and requires TLS 1.2 (not supported by all openssl versions)